scan-command/src/test/java/io/ballerina/scan/internal/ScanCmdTest.java

```diff
@@ -337,21 +342,32 @@ Object[][] rulesProvider() {
         return new Object[][] {
                 {
                     new ArrayList<>(List.of(
-                            RuleFactory.createRule(1, "rule 1", RuleKind.CODE_SMELL, BALLERINA_ORG, "exampleModule"),
-                            RuleFactory.createRule(3, "rule 3", RuleKind.BUG, BALLERINAX_ORG, "exampleModule"),
-                            RuleFactory.createRule(2, "rule 2", RuleKind.VULNERABILITY, "wso2", "exampleModule"),
+                            RuleFactory.createRule(1, "rule 1", RuleKind.CODE_SMELL,
```

Comment:
I would leave as is and only break those that need to be broken. Easier to read.